examples/common.cpp

```diff
@@ -31,6 +33,43 @@ extern "C" __declspec(dllimport) int __stdcall WideCharToMultiByte(unsigned int
 #define CP_UTF8 65001
 #endif
 
+void split_args(std::string args_string, std::vector<std::string> & output_args) 
```

Comment:
Could be possibly rewritten with `std::quoted`, but requires C++14.

Reply:
I think should use const std::string &args_string